gamestonk_terminal/portfolio/README.md

```diff
@@ -0,0 +1,40 @@
+#PORTFOLIO
+
+This was developed to interface with Robinhood.  
+
+Once this screen is accessed, the first command to be run is 
+````
+login
+````
+Which will use your robinhood login information, which should be stored as environment variables or in the config_terminal.py
+
+Currently there are 2 options:
+* [hold](#hold)
+    * Look at current (stock only) holdings
+* [rhhist](#rhhist)
+    * Get and plot rhhistorical portfolio
+
+## hold <a name="hold"></a>
+
+Displays current holdings to the console:
+
+````
+usage: hold
+````
+There are no additional flags.  This just prints all current stonks, their last price, previous close, your equity and
+the % change from previous close.
+
+Example:
```

Comment:
You need to add a spacing between images and the text, so that they don't appear in same line.

Also, I believe the other day I told you that you need to create a new issue for it, but I think you can literally drag and drop it to README and it works.

Reply:
When I drag in a photo into my pycharm, it doesn't do anything, I just found that the upload issue works very simply.

Reply:
Yea, I usually don't do that bit locally. I do it after pushing my branch to remote, sometimes even before creating the PR.

E.g. once you push, and create a PR, at the top you'll see:
`jmaslek:robinhood`
if you create there it will take you to your branch on remote through web browser. Then you can access the readme document, edit it and drag+drop a figure there. This also allows you to preview what it would look like.

Then when you commit through web browser, this will be updated on the remote. In order for you to get that exact same commit locally, you'll want to do git pull.  If you don't do this last bit and commit something locally you'll get in this weird state where your local branch is 1 commit behind remote but also 1 commit ahead.

Hope this makes some sense